src/nectarbase/signedtransactions.py

```
@@ -35,10 +35,10 @@ def add_custom_chains(self, custom_chain):
                 if c not in self.known_chains:
                     self.known_chains[c] = custom_chain[c]
 
-    def sign(self, wifkeys, chain="STEEM"):
+    def sign(self, wifkeys, chain="HIVE"):
         return super(Signed_Transaction, self).sign(wifkeys, chain)
 
-    def verify(self, pubkeys=[], chain="STEEM", recover_parameter=False):
+    def verify(self, pubkeys=[], chain="HIVE", recover_parameter=False):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)